elastic/kibana
# [Index Mgmt] Fix the count of docs per index

## Summary

Closes https://github.com/elastic/kibana/issues/130851

**Testing**:
 - Index 33 documents across 3 indices. Use 1 replica shard for settings of the indices.
 - Verify the indices show as green and the document counts are correct in the overview table.
 - Delete 6-7 documents from one of the indices, and [refresh](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-refresh.html) the index.
 - Verify the new document count is correct, and the number of deleted documents is correct under the detail view of the index

Release note: Fixed a bug in Index Management where the number of documents for an index could appear wrong.

Checklist:
 - [X] Verify if count of deleted documents needs the same fix

### Screenshots
Before: left
After: right

![image](https://user-images.githubusercontent.com/908371/177623269-e1ce9980-d54c-4358-a1d5-0b0969a1c9c3.png)
![image](https://user-images.githubusercontent.com/908371/177623321-a5a61787-ed82-45c2-b713-05303ad6e275.png)
